.changeset/silent-ducks-wait.md

```diff
@@ -0,0 +1,6 @@
+---
+'@clerk/clerk-js': patch
+'@clerk/types': patch
+---
+
+Remove `virtual` routing option from components.
```

Comment:
We're not really deleting it but removing it from the public API surface. So maybe something like this?

```suggestion
Remove `'virtual'` from the `routing` option. The `'virtual'` value is only used internally and should not be part of the public API.
```